src/Tasks.UnitTests/Exec_Tests.cs

```diff
@@ -103,8 +103,8 @@ public void ExitCodeCausesFailure()
         [Fact]
         public void Timeout()
         {
-            // On non-Windows the exit code of a killed process is SIGTERM (143)
-            int expectedExitCode = NativeMethodsShared.IsWindows ? -1 : 143;
+            // On non-Windows the exit code of a killed process is SIGKILL (137)
+            int expectedExitCode = NativeMethodsShared.IsWindows ? -1 : 137;
```

Comment:
will this break Mac?

Reply:
This works on Mac also. By convention exit code when terminated by signal are calculated as `128 + signo`.
The signal number for `SIGKILL` is `9` on any OS I know.
Otherwise these guys need to update their lyrics: https://www.youtube.com/watch?v=Fow7iUaKrq4 :smile:.